src/base/preferences.cpp

```diff
@@ -1330,6 +1330,19 @@ void Preferences::setMarkOfTheWebEnabled(const bool enabled)
     setValue(u"Preferences/Advanced/markOfTheWeb"_s, enabled);
 }
 
+bool Preferences::isIgnoreSSLErrors() const
+{
+    return value(u"Preferences/Advanced/IgnoreSSLErrors"_s, false);
```

Comment:
Maybe it makes sense to keep the existing default behavior (i.e. ignoring SSL errors) in v5.0, and change it only in next major update (v5.1)?

Reply:
This is a security fix* essentially. I am hesitant in keeping it off.

*theoretically users could be fed malicious program-update URLs and only get a log warning

Reply:
Ok, suit yourself.